src/ui/blocks/RemoteSource.hpp

```diff
@@ -59,41 +62,72 @@ struct RemoteStreamSource : public gr::Block<RemoteStreamSource<T>> {
             } else {
                 std::ranges::transform(in, output.begin() + written, [](float v) { return static_cast<T>(v); });
             }
+            {
+                const auto  now     = std::chrono::duration_cast<std::chrono::nanoseconds>(std::chrono::high_resolution_clock::now().time_since_epoch());
+                auto        latency = now - std::chrono::nanoseconds(d.acq.acqLocalTimeStamp.value());
+                std::size_t idx     = written + in.size() - 1;
+                fmt::print("RemoteSource: url:{}, event[{}]: CHUNK_END ({}), latency against localtime: {}\n", remote_uri, idx, d.acq.acqLocalTimeStamp.value(), latency);
```

Comment:
debug printout?!? Becomes a bit verbose when testing and latency is not a concern.